### ISSUE TITLE
XSParseInfix.h: No such file or directory

### ISSUE THREAD

**Issue by eserte** [Leont/syntax-infix-smartmatch#3]
The compilation fails on some of my smoker systems:
```
...
require ExtUtils::ParseXS; ExtUtils::ParseXS::process_file('filename','lib/Syntax/Infix/Smartmatch.xs','prototypes',0,'output','lib/Smartmatch.c')
cc -I. -Ilib/Syntax/Infix -Iinclude -I/opt/perl-5.38.2D/lib/5.38.2/aarch64-linux-ld/CORE -DVERSION="0.006" -DXS_VERSION="0.006" -fPIC -I. -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -O2 -o Smartmatch.o -c lib/Smartmatch.c
lib/Syntax/Infix/Smartmatch.xs:8:10: fatal error: XSParseInfix.h: No such file or directory
    8 | #include "XSParseInfix.h"
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
Could not run command cc -I. -Ilib/Syntax/Infix -Iinclude -I/opt/perl-5.38.2D/lib/5.38.2/aarch64-linux-ld/CORE -DVERSION="0.006" -DXS_VERSION="0.006" -fPIC -I. -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -O2 -o Smartmatch.o -c lib/Smartmatch.c at /opt/perl-5.38.2D/lib/site_perl/5.38.2/ExtUtils/Builder/Action/Command.pm line 36.
...
```

**Comment by Leont** [comment]
That doesn't make sense. Does upgrading Dist::Build make it disappear?

**Comment by Leont** [comment]
I see the problem, I didn't specify the dependency for `XS::Parse::Infix` on the right version. Fixed that in 0.007.
